src/libtorch.cc

```diff
@@ -350,7 +351,8 @@ ModelState::ParseParameters()
         TRITONSERVER_ErrorDelete(err);
       }
     } else {
-      enable_nvfuser_pair_ = {true, enable_nvfuser};
+      // Override, disable NvFuser till fixed
+      enable_nvfuser_pair_ = {true, false /* enable_nvfuser */};
```

Comment:
Shouldn't we atleast emit a warning saying the nvfuser is being disabled instead of silently disabling it?

Reply:
Corrected to update enable_nvfuser, that way the log in L357 will accurately reflect if the NvFuser has been disabled.

Reply:
@tanmayv25 would you rather this be a Warning or an Info log?

Reply:
I would prefer an  additional warning. This way user would get to know why their preferred configuration were not selected.

Reply:
It probably doesn't make sense to have two logs so I am changing the current Log to warning for now.